src/components/link/Link.ts

```diff
@@ -13,6 +15,16 @@ class Link extends BaseBlock {
 
     protected componentDidMount(): void {
         this._props.class = `${this._props.class} base-link`;
+        const link = this.getContent() as HTMLLinkElement;
```

Comment:
Здесь можно сделать такую же штуку, как и с querySelector. Это не обязательно, но если получится - будет очень  круто
```suggestion
        const link = this.getContent<HTMLLinkElement>();
```